[WF] Update: Optimize about page (#26)
bitsandvolts/bitsandvolts.github.io: _pages/about.md

toc_icon: "align-justify"
---

> # What you can expect

Reviewer: If this is the new first section title, it doesn't fit well anymore, IMO. Either we could leave it out, or use a different title.

Author: What would you prefer? Something like "Bits&Volts - A blog for everyone"?

Reviewer: Sounds good.

There are already tons of good resources available. So you are probably
wondering why we are starting another blog. Back when we were fresh students, we
always felt that the barrier to enter the field of electronics is way higher

> than it should be. Even at university, we learned deep and complex theory, while

Reviewer: Past tense would fit better, I think:
```suggestion
than it should've been. Even at university, we learned deep and complex theory, while
```

> simple and practical aspects were never explained. So many times we did things

Reviewer: "Never" sounds a bit too "hard", in my view.
```suggestion
simple and practical aspects were rarely explained. So many times we did things
```

wrong or inefficiently while a voice in our head asked: "Why didn't anyone tell
us?". The tricky part as a beginner is that most of the time you don't know what

> to look for or where to look. How can you distinguish between a good and a bad

Reviewer: ```suggestion
to look for, or where to look. How can you distinguish between a good and a bad
```

Author: are you sure?

Reviewer: Not anymore.

Author: I changed that part a bit

> resource? Often we come across blogs that either do only cover a small piece of

Reviewer: I like that part now.

Also, I wouldn't reduce this to just blogs, maybe use also "resources" here.

toc_icon: "align-justify"
---

# Bits&Volts - A blog for everyone

There are already tons of good resources available. So you are probably
wondering why we are starting another blog. Back when we were fresh students, we
always felt that the barrier to enter the field of electronics is way higher
than it should've been. Even at university, we learned deep and complex theory,
while simple and practical aspects were rarely explained. So many times we did
things wrong or inefficiently while a voice in our head asked: "Why didn't
anyone tell us?". It can be hard to get going with electronics since you do not
know the right place to start. How can you distinguish between a good and a bad

> resource? Often we come across blogs or other kinds of resources that either do

Reviewer: Hm, I'm still not a fan of "blog" here. Resource would be enough, IMO.

Unless you know of a good example real world for a "bad blog".

But if you think that it is fine, leave it.

Author: I tried to avoid resource since it is the last word of the previous sentence. Any other idea?

Reviewer: I think the repetition is acutally not that bad here. It is like if the second sentence picks up "resource" from the previous one and then elaborates on it.
```suggestion
resource? Often we come across resources that either do ...
```

> only cover a small piece of the whole pie or leave out too much detail, making

Reviewer: Aren't these two (small piece of pie; leave out detail) more or less the same?

Author: Not really. 
1) cover a small part very detailed
2) cover everything not that detailed

Reviewer: Ah ok. But to be honest, I didn't fully get it the first time I read it.
